### PR TITLE
fix: [oadp-1.1] CRD description of default restore resource priorities

### PR DESCRIPTION
**Pull request by mateusoliveira43**
Update CRD description and follow refactor from Scott PR's https://github.com/openshift/oadp-operator/pull/1147

### CODE REVIEW

File: controllers/velero_test.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ const (
 	proxyEnvValue                          = "http://proxy.example.com:8080"
 	argsMetricsPortTest              int32 = 69420
 	defaultFileSystemBackupTimeout         = "--restic-timeout=1h"
-	defaultRestoreResourcePriorities       = "--restore-resource-priorities=securitycontextconstraints,customresourcedefinitions,namespaces,storageclasses,volumesnapshotbackups.datamover.oadp.openshift.io,volumesnapshotclass.snapshot.storage.k8s.io,volumesnapshotcontents.snapshot.storage.k8s.io,volumesnapshots.snapshot.storage.k8s.io,datauploads.velero.io,persistentvolumes,persistentvolumeclaims,serviceaccounts,secrets,configmaps,limitranges,pods,replicasets.apps,clusterclasses.cluster.x-k8s.io,endpoints,services,-,clusterbootstraps.run.tanzu.vmware.com,clusters.cluster.x-k8s.io,clusterresourcesets.addons.cluster.x-k8s.io"
```
Comment (mateusoliveira43) [inline controllers/velero_test.go:34]: as we talked, `datauploads.velero.io` is just for OADP 1.3/Velero 1.12, right?

Comment (sseago) [reply controllers/velero_test.go:34]: @mateusoliveira43 That's right.

**Review by sseago — APPROVED** [review]

**Review by shubham-pampattiwar — APPROVED** [review]

**Comment by mateusoliveira43** [comment]
1.1.6 was released
/unhold

**Pull request merged**